unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -165,6 +165,20 @@ public UTF8String toLowerCase() {
     return UTF8String.fromString(toString().toLowerCase());
   }
 
+  public UTF8String initCap(final byte[] byteArr) {
+    if (byteArr == null) return null;
+    if (byteArr[0] <= 122 && byteArr[0] >= 97) //a-z
```

Comment:
I don't think that we can rely on this range. There are a lot of european languages with special characters like `ä and Ä ü and Ü ö and Ö` and all the french stuff `è and È`.